### PR TITLE
Add IPHONEOS_DEPLOYMENT_TARGET & Update README.md

### PR DESCRIPTION
**Pull request by sbarow**
When using `0.2.0` release with `Carthage` the user will receive an error, this diff updates the version to reflect the latest release.

```
*** Skipped building swift-concurrency due to the error:
Dependency "swift-concurrency" has no shared framework schemes for any of the platforms: iOS
```

Tested with `0.3.0` and works correctly with `Carthage`

Added `IPHONEOS_DEPLOYMENT_TARGET` and set it to `8.0.0` (this version should be fine right?)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ $ brew install carthage
 To integrate Swift-Concurrency into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "https://github.com/uber/swift-concurrency.git" ~> 0.2.0
+github "https://github.com/uber/swift-concurrency.git" ~> 0.3.0
```
Comment (neakor) [inline README.md]: Can we update to version 0.4.0 please?

**Comment by sbarow** [comment]
@neakor done.

**Comment by neakor** [comment]
Thank you!

**Comment by sbarow** [comment]
@neakor I don't have write access to this repo, so you'll have to do the merge :-)

**Comment by neakor** [comment]
Oh sorry haha! Thanks again.

**Pull request merged**